sympy/printing/fcode.py

```diff
@@ -409,7 +409,7 @@ def split_pos_code(line, endpos):
                 if pos == 0:
                     return endpos
             return pos
-        # split line by line and add the splitted lines to result
+        # split line by line and add the split lines to result
```

Comment:
Is this change correct?

Reply:
@asmeurer https://www.quora.com/Why-is-the-word-splitted-not-grammatically-correct